src/mozanalysis/metrics/__init__.py

```diff
@@ -185,10 +185,23 @@ class Metric:
     """Represents an experiment metric.
 
     Needs to be combined with an analysis window to be measurable!
+
+    Args:
+        name (str): A slug; uniquely identifies this metric in tables
+        data_source (DataSource): where to find the metric
+        select_expr (str): a SQL snippet representing a clause of a SELECT
+            expression describing how to compute the metric; must include an
+            aggregation function since it will be GROUPed BY client_id
+            and branch
+        friendly_name (str): A human-readable dashboard title for this metric
+        description (str): A paragraph of Markdown-formatted text describing
+            what the metric measures, to be shown on dashboards
     """
     name = attr.ib(type=str)
     data_source = attr.ib(type=DataSource)
     select_expr = attr.ib(type=str)
+    friendly_name = attr.ib(type=str, default="")
```

Comment:
I'm a little uneasy about defaulting to "" rather than `None`. The latter better represents that no value was provided. I guess it's unlikely that someone will try to key a dictionary with `friendly_name` or count distinct `friendly_name` when `name` is available (in which case defaulting to `None` would save grief), but it's possible?

So I have a mild preference for changing this and description to default to `None`. If it's annoying to handle the `None` values downstream then I can live with the existing "".

Reply:
Oh, yeah, None is a fine choice here, idk why I chose `""`.